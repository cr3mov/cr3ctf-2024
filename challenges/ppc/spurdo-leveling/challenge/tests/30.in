116472547747644400 902246344381955230
860453 8214543756
860453 7668588599
