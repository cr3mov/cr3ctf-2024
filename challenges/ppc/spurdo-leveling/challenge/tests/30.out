41061032352571 49469634 49469634
